tests/manage/monitoring/pagerduty/test_deployment_status.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import tier4, tier4a
+from ocs_ci.ocs import constants
+from ocs_ci.utility import pagerduty
+
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
+@pytest.mark.polarion_id("OCS-1052")
```

Comment:
skip if not managed ODF?

Reply:
Added `skipif_not_managed_service` marker